README.md

```diff
@@ -53,16 +53,19 @@ brew info autoenv
 You can then copy the `.env_SAMPLE` file to `.env`, then update any environment variables accordingly.
 
 ### Dependencies
-This project uses `requirements.txt` files for defining dependencies, so you
-can get up and running with `pip`:
+First, create a virtual environment for Python dependencies:
+```
+mkvirtualenv ccdb5-api
+```
 
-```shell
-$ pip install -r requirements.txt       # modules required for execution
+Next, use `pip` to install dependencies, which are defined in `setup.py`:
+```
+pip install -e '.[testing]
```

Comment:
Minor typo: missing trailing `'` to complete the command.

Reply:
Ah, thank you. Fixed.